UI/About.h

```diff
@@ -24,6 +27,7 @@ class About : public CUIWnd {
 
 private:
     void DoLayout();
+    void LoadLicense(const boost::filesystem::path& licensePath);
```

Comment:
probably doesn't need to be in the header... just in an anonymous namespace in the .cpp

Reply:
moved to anon. ReadFile